.github/workflows/ci.yml

```diff
@@ -2,9 +2,8 @@ name: Run Linter and Tests
 
 on:
   pull_request:
-  push:
-    branches: [ main ]
   workflow_call:
+  workflow_dispatch:
```

Comment:
i am _hoping_ the addition of this allows me to call the workflow from a particular branch instead of creating a PR according to [this documentation](https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow), but it didn’t work adding it to this PR and i’m hoping it does once merged into main.

Reply:
🤔 i believe things will fail since we’re running `bundle exec rspec` and that will include capybara so we should be okay